transformers/integration_tests/integration_tests_suite_test.go

```diff
@@ -24,26 +24,22 @@ var (
 )
 
 var _ = BeforeSuite(func() {
-	testConfig := viper.New()
-	testConfig.SetConfigName("testing")
-	testConfig.AddConfigPath("$GOPATH/src/github.com/makerdao/vdb-mcd-transformers/environments/")
-	err := testConfig.ReadInConfig()
-	Expect(err).To(BeNil())
-	ipc = testConfig.GetString("client.ipcPath")
+	ipc = viper.GetString("client.ipcPath")
 	// If we don't have an ipc path in the config file, check the env variable
 	if ipc == "" {
-		configErr := testConfig.BindEnv("url", "CLIENT_IPCPATH")
+		configErr := viper.BindEnv("url", "CLIENT_IPCPATH")
 		Expect(configErr).To(BeNil(), "Unable to bind url to CLIENT_IPCPATH env var")
-		ipc = testConfig.GetString("url")
+		ipc = viper.GetString("url")
 	}
-	Expect(ipc).NotTo(BeEmpty(), "testing.toml IPC path or $CLIENT_IPCPATH env variable need to be set")
+	Expect(ipc).NotTo(BeEmpty(), "$CLIENT_IPCPATH env variable need to be set")
 
 	rpcClient, ethClient, clientErr := getClients(ipc)
 	Expect(clientErr).NotTo(HaveOccurred())
 	var blockChainErr error
 	blockChain, blockChainErr = getBlockChain(rpcClient, ethClient)
 	Expect(blockChainErr).NotTo(HaveOccurred())
 
+	// the init function in test_config set the config files for the test database and transformers
```

Comment:
Clarifying the above, wondering if it would make sense to also have something like `blockchain = test_config.NewTestBlockchain()` that uses the `ipc` we have as a result of calling `init` in the `test_config`

Reply:
Though maybe that's a bad idea if it means we'd fail silently if there were no ipc path? 🤔